### PR TITLE
Move user related calls to UserController

### PR DESCRIPTION
**Pull request by lakhina**

### CODE REVIEW

File: web-gateway/app/controllers/Main.java [hunk 1]
```diff
@@ -38,36 +38,4 @@ public Main(Configuration config, MessagesApi messagesApi, UserService userServi
         );
     }
```
Comment (TimMoore) [inline web-gateway/app/controllers/Main.java]: The `userService` field is no longer used, and can be removed.

Comment (TimMoore) [inline web-gateway/app/controllers/Main.java]: Same with `formFactory` and `showInlineInstruction`.

File: web-gateway/app/controllers/UserController.java [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+
+import javax.inject.Inject;
+import java.time.Duration;
+import java.time.Instant;
+import java.time.temporal.ChronoUnit;
+import java.util.*;
+import java.util.concurrent.CompletableFuture;
+import java.util.concurrent.CompletionStage;
+
+import static com.example.auction.security.ClientSecurity.authenticate;
```
Comment (TimMoore) [inline web-gateway/app/controllers/UserController.java]: A lot of these `import` statements aren't actually used in the code. Recommend using an IDE function to automatically organize your imports:

- https://www.jetbrains.com/help/idea/optimizing-imports.html
- https://help.eclipse.org/neon/topic/org.eclipse.jdt.doc.user/gettingStarted/qs-OrganizeImports.htm?cp=1_1_0_3_6

You may need to customize the settings for import order to make sure that the imports don't flip-flop back and forth when different people use different IDEs to edit the file.

Here are the settings in my IDE (the defaults for IntelliJ IDEA):

screen shot 2017-06-22 at 12 01 39 pm

File: web-gateway/app/controllers/Main.java (hunk 1 above)
Comment (lakhina) [inline web-gateway/app/controllers/Main.java]: @TimMoore I removed formFactory and showInlineInstruction but userService need to be there even for index page request.

Comment (TimMoore) [inline web-gateway/app/controllers/Main.java]: It needs to be injected into the constructor and passed to `super`, but I don't see anywhere the instance variable is read.

**Comment by TimMoore** [comment]
@lakhina please let us know what your plans are with this. Can you do the clean up I suggested in my comments? IDEA should be able to detect and auto-fix most of these problems. Let me know if you need more specific advice.

**Comment by lakhina** [comment]
@TimMoore I have just optimised the imports in UserController.java Please review and let me know if any changes required

File: web-gateway/app/controllers/Main.java (hunk 1 above)
Comment (TimMoore) [inline web-gateway/app/controllers/Main.java]: What about the suggestion to remove the unused fields from `Main`? IDEA should highlight these, too. F2 will jump between problems that IDEA notices. Alt-Enter will provide options to fix them.

Comment (lakhina) [inline web-gateway/app/controllers/Main.java]: @TimMoore  userService can't be removed as it is needed in the index page.

Comment (TimMoore) [inline web-gateway/app/controllers/Main.java]: Sorry... I had missed the changes to this file in your most recent commit. I thought only `UserController.java` had changed. Your most recent version is exactly what I had in mind, thanks!

**Pull request merged**